content/en/docs/languages/java/automatic/spring-boot.md

```diff
@@ -144,9 +143,305 @@ dependencies {
 
 {{% /tab %}} {{< /tabpane>}}
 
-### Additional instrumentations
+### OTLP Exporter
+
+This package provides auto configurations for
+[OTLP](https://github.com/open-telemetry/opentelemetry-java/tree/main/exporters/otlp)
+and
+[Logging](https://github.com/open-telemetry/opentelemetry-java/tree/main/exporters/logging)
+Span Exporters.
+
+#### Exporter Properties
+
+| Feature       | Property                    | Default Value    |
+| ------------- | --------------------------- | ---------------- |
+| Otlp Exporter | otel.exporter.otlp.endpoint | `localhost:4317` |
+|               | otel.exporter.otlp.protocol | `http/protobuf`  |
+|               | otel.exporter.otlp.headers  |                  |
+|               | otel.exporter.otlp.timeout  | `1s`             |
+
+The `otel.exporter.otlp.headers` property can be specified as a comma-separated
+list, which is compliant with the
+[specification](https://opentelemetry.io/docs/concepts/sdk-configuration/otlp-exporter-configuration/#otel_exporter_otlp_headers).
```

Comment:
Site-local pages should be referenced using a path:

```suggestion
[specification](/docs/concepts/sdk-configuration/otlp-exporter-configuration/#otel_exporter_otlp_headers).
```

Make this change here and elsewhere.